tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -107,7 +107,8 @@ function main() {
   mkdir -p ${TMPDIR}/third_party
   pushd ${RUNFILES%org_tensorflow}
   for header in $(find protobuf -name \*.h); do
-    cp --parents "$header" ${TMPDIR}/google;
+    mkdir -p "${TMPDIR}/google/$(dirname ${header})"
+    cp "$header" "${TMPDIR}/google/$(dirname ${header})/"
```

Comment:
I think calling dirname at least once (in the first newly added line) is necessary. But gunan@ is right that the second new line doesn't need it. It can just be something like,
cp "$header" "${TMPDIR}/google/${header}"

But it's not a big deal. I'll approve the PR as is.